src/gateways/MsSQL/resetPassword.js

```diff
@@ -0,0 +1,55 @@
+import bcrypt from "bcryptjs";
```

Comment:
Could you add a unit test for this gateway please?

Reply:
Unit test added